strider/util.py

```diff
@@ -87,6 +110,17 @@ def predicate_inverse(self, predicate):
             return self.old_case_to_new_case(predicate_inverse_old_format)
         return None
 
+    def entity_prefixes(self, entity):
+        """ Get prefixes for a given entity """
+        old_format = self.new_case_to_old_case(entity)
+        element = self.bmt.get_element(old_format)
```

Comment:
How fast is this? We could memoize this function if it's helpful. Then we'd have a map pretending to be a function pretending to be a map!